apps/qrcode-generator/static/style.css

```diff
@@ -0,0 +1,13 @@
+#urlPara {
```

Comment:
Consider renaming ids in the CSS file; short, all letters lower case and use of `-` to join words.

Reply:
Done